index.html

```diff
@@ -1,51 +1,92 @@
 <!DOCTYPE html>
 <html lang="pl">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="reset.css">
-    <link rel="stylesheet" href="style.css">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap" rel="stylesheet">
-    <title>My-Osom-Site</title>
-</head>
-<body>
-    <header>
-        <div class="nav">
-            <img src="logo.png" alt="logo">
-            <ul>
-                <li><a href="#">Home</a></li>
-                <li><a href="About/index.html" target="_blank">About</a></li>
-                <li><a href="Contact/index.html" target="_blank">Contact</a></li>
-            </ul>
-        </div>
-        <hr>
-    </header>
-    <main>
-        
-            <div class="hero">
-               <h1>My Osom Site</h1>
-            </div>
-       
-        <div class="blog">
-            <article class="article">
-                <h2>my first article</h2>
-                <p id="text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec id nibh sit amet tortor rhoncus pharetra non fermentum quam. Nam dui libero, ultricies a mi vitae, convallis congue felis. Aliquam erat volutpat. In sit amet faucibus tellus, vel semper justo. Fusce ullamcorper pharetra iaculis. Donec venenatis porta dolor, id viverra.</p>
-                <p id="date">20.02.2020</p>
-            </article>
-            <article class="article">
-                <h2>my second article</h2>
-                <p id="text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec id nibh sit amet tortor rhoncus pharetra non fermentum quam. Nam dui libero, ultricies a mi vitae, convallis congue felis. Aliquam erat volutpat. In sit amet faucibus tellus, vel semper justo. Fusce ullamcorper pharetra iaculis. Donec venenatis porta dolor, id viverra.</p>
-                <p id="date">24.04.2020</p>
-            </article>
-        </div>
-        <hr>
-    </main>
-    <footer>  
-        <p>Coded by: Wiktor Bonderek</p>
-        <p>wiktor.bonderek@shoper.pl</p>
-    </footer>
-</body>
+ <head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- <link rel="stylesheet" href="reset.css"> -->
+  <link rel="stylesheet" href="style.css">
+  <title>Lista zakupów</title>
+ </head>
+ <body>
+  
+   <header>
+     <hr>
+     <h1>Lista zakupów</h1>
+     <span>
+      <hr>
```

Comment:
tak nie robimy :) jeżeli chcesz mieć wizualnie 2 linie korzystamy z `cssa` fakt, wiekszosc przegladarek narysuje ten tag jako linie, ale uzywamy go tylko do celów semantycznych, jak cos oddziela np 2 paragrafy, pamietajmy ze html to semantyka a css jest do warstwy wizualne, tutaj należało skorzystać z bordera